plugins/webp-uploads/helper.php

```diff
@@ -16,20 +16,24 @@
  * for example an image/jpeg can be converted into an image/webp.
  *
  * @since 1.0.0
+ * @since n.e.x.t Added support for AVIF.
  *
  * @return array<string, array<string>> An array of valid mime types, where the key is the mime type and the value is the extension type.
  */
 function webp_uploads_get_upload_image_mime_transforms() {
+
+	// Check the selected output format.
+	$output_format = get_option( 'perflab_generate_avif_and_webp', wp_image_editor_supports( array( 'mime_type' => 'image/avif' ) ) ?  'avif' : 'webp' );
+
 	$default_transforms = array(
-		'image/jpeg' => array( 'image/webp' ),
-		'image/webp' => array( 'image/webp' ),
```

Comment:
self format is the default so i removed this line

Reply:
well, I changed it to `'image/jpeg' => array( 'image/' . $output_format )` which is now either webp or avif. 

Also, I have since restored these lines because having the self mapping actually changed the behavior - the plugin doesn't create the root source mapping if only one transform is included.